flask_monitoringdashboard/database/versions.py

```diff
@@ -21,7 +21,8 @@ def get_versions(db_session, end=None):
     :param end: the versions that are used in a specific endpoint
     :return: a list with the versions (as a string)
     """
-    result = db_session.query(distinct(FunctionCall.version)). \
+    result = db_session.query(FunctionCall). \
```

Comment:
This method now fails a test

Reply:
Hi, i think it should be OK to pass the unittest now XD